README.md

```diff
@@ -1,235 +1,112 @@
-  - [Deployment](#triangular_flag_on_post-deployment)
+    - [Setup](#setup)
+    - [Prerequisites](#prerequisites)
+    - [Install](#install)
+    - [Usage](#usage)
 - [👥 Authors](#authors)
-- [🔭 Future Features](#future-features)
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
+# 📖 Portfolio <a name="about-project"></a>
 
-**[your_project__name]** is a...
+> My name is **Abdullah al mamun**, i am a software developer.
+> This will showcase my accomplishment, skill and my recent work.
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
+> This project built with html & css.
 
-<details>
-  <summary>Client</summary>
-  <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
-  </ul>
-</details>
-
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
-
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
-  </ul>
-</details>
+- HTML
+- CSS
+- Flexbox
 
 <!-- Features -->
 
-### Key Features <a name="key-features"></a>
-
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link](https://dev-mamun.github.io/microverse-portfolio/)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
 To get a local copy up and running, follow these steps.
 
 ### Prerequisites
 
 In order to run this project you need:
-
-<!--
-Example command:
-
-```sh
- gem install rails
-```
- -->
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>Html & CSS</li>
+    <li><a href="https://github.com/microverseinc/linters-config/tree/master/html-css">Linter</a></li>
+  </ul>
+</details>
 
 ### Setup
 
 Clone this repository to your desired folder:
 
-<!--
-Example commands:
-
 ```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
+  git clone git@github.com:dev-mamun/microverse-portfolio.git
 ```
---->
+
 
 ### Install
 
 Install this project with:
 
-<!--
-Example command:
-
-```sh
-  cd my-project
-  gem install
-```
---->
-
-### Usage
-
-To run the project, execute the following command:
-
-<!--
-Example command:
-
 ```sh
-  rails server
-```
---->
-
```

Comment:
- [ ] Please make you add the getting started section and make sure it is more descriptive and the content should be the steps to follow to get your project up and running on a local machine. 

For example,
```
## Getting started

You can add the steps to follow to clone your project,

### Prerequisites
if you don't have git installed on your machine you can download it from [here](https://git-scm.com/downloads).

### Setup
Once you have git installed on your machine you can clone your project by running the command below to clone your project to your local machine


`git clone <your project link>`

Alternatively, you can download the zip file of your project by clicking on the `Code` button on the right side of your project page and clicking on `Download ZIP`

### Install
Install this project with:

``sh
  cd my-project
``

for this project, you don't need to install any dependencies but if you prefer to install any dev dependencies run:
``
npm install
``

### Usage

To run the project,

Locate the folder where you cloned your project and open the `index.html` file in your browser to view your project.
```

Reply:
@Owusu-Desmond done